pubspec.yaml

```diff
@@ -7,12 +7,17 @@ environment:
   sdk: ^3.1.1
 
 dependencies:
+  code_builder: ^4.7.0
+  dart_style: ^2.3.3
+  geojson: ^1.0.0
+  geopoint: ^1.0.0
+  latlong2: ^0.8.2
+  point_in_polygon: ^1.0.0
```

Comment:
I'm unsure whether this works reliable for geo-coordinates, since they can "wrap around" the earth.

I think using a package like geodesy is better. See this function: https://pub.dev/documentation/geodesy/latest/geodesy/Geodesy/isGeoPointInPolygon.html